utils/travis/steps/install.sh

```diff
@@ -8,12 +8,15 @@ export PLAY_TEST=true
 export MP_TEST=true
 export WML_TEST_TIME=15
 export BOOST_TEST=true
+export LTO=false
 
 if [ "$OPT" = "-O0" ]; then
     export EXTRA_FLAGS_RELEASE="-O0"
     export PLAY_TEST=false
     export MP_TEST=false
     export WML_TEST_TIME=20
+else
+    export LTO=false
```

Comment:
Is this line here in order to allow us to enable LTO for optimized builds by changing it?

If so, please add a comment to explain it.

Reply:
Added.